vcd/resource_vcd_external_network_v2.go

```diff
@@ -152,12 +162,12 @@ func resourceVcdExternalNetworkV2() *schema.Resource {
 				Elem:        networkV2VsphereNetwork,
 			},
 			"nsxt_network": &schema.Schema{
-				Type:         schema.TypeSet,
+				Type:         schema.TypeList,
 				Optional:     true,
 				ExactlyOneOf: []string{"vsphere_network", "nsxt_network"},
 				MaxItems:     1,
 				ForceNew:     true,
-				Description:  "Reference to NSX-T Tier-0 router and manager",
+				Description:  "Reference to NSX-T Tier-0 router or Segment and manager",
```

Comment:
Nit: does "Segment" deserve being capitalized? Alternatively, should "Router" and "Manager" be too (though then it becomes a question for all fields' descriptions :) )?

Reply:
Fixed. Made it lower case here.